packages/pages/src/bundler.js

```diff
@@ -79,3 +75,11 @@ try {
 } catch (e) {
   console.error(e);
 }
+
+/**
+ * Copy the sh binary.
+ */
+copyFileSync(
+  path.resolve("./src/bin/pages.sh"),
+  path.resolve("./dist/bin/pages.sh")
```

Comment:
use pluginOutputPath?

Reply:
pluginOutputPath is a different path than the bin path